operations/research-and-development/product/technical-writing-team-handbook/README.md

```diff
@@ -8,24 +8,21 @@ The Mattermost Technical Writers focus primarily on writing, editing, and mainta
 
 Documentation and technical writing at Mattermost covers \(but isn't limited to\) the following verticals:
 
-* Messaging
-  * [Mattermost Overview](https://docs.mattermost.com/overview/index.html)
-  * [User's Guide](https://docs.mattermost.com/guides/user.html)
-  * [Administrator's Guide](https://docs.mattermost.com/guides/administrator.html)
+* Channels
   * Mattermost Cloud
-  * Apps Framework and Marketplace
   * Self-managed deployments
   * Messaging features and functionality
-* Incident Collaboration
-* Focalboard
+* Playbooks
+* Boards
 * Growth
   * Customer Portal
   * Telemetry
+* Apps Framework and Marketplace
```

Comment:
Although we often call it the Apps Framework internally, I think we landed on App Framework for external advertising.

```suggestion
* App Framework and Marketplace
```